packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -463,6 +464,84 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
     });
   }
 
+  @override
+  Future<void> handlePointerEventPack(List<PointerEventPack> records) {
+    assert(records != null);
+    assert(records.isNotEmpty);
+    // hitTestHistory is an equivalence of _hitTests in [GestureBinding]
+    final Map<int, HitTestResult> hitTestHistory = <int, HitTestResult>{};
+    return TestAsyncUtils.guard<void>(() async {
+      final DateTime startTime = binding.clock.now();
+      for(final PointerEventPack pack in records) {
+        final Duration timeDiff = pack.timeStamp - binding.clock.now().difference(startTime);
```

Comment:
This seems to suggest that `pack.timeStamp` is relative to the start of `handlePointerEventPack`. It might be helpful to clearly document it in `PointerEventPack.timeStamp`.